### ISSUE TITLE
Push to Maven Repository

### ISSUE THREAD

**Issue by mikulucky** [francisrangel/rest-mock#2]
This seems to be a great mocking framework for writing an API wrapper against. Could you possibly push this to a maven repository so it can be pulled into other projects easily?

**Comment by francisrangel** [comment]
@mikulucky yes, I will take a look pushing it to maven repo. I will let you know the news here.

**Comment by gb** [comment]
@mikulucky a decade later, this was finally resolved. Please feel free to try:

```
<dependency>
  <groupId>org.simulatest</groupId>
  <artifactId>restmock</artifactId>
  <version>0.1.0</version>
  <scope>test</scope>
</dependency>
```

Cheers!
